### ISSUE TITLE
2.0.1.18: FTP Backup Test Connection fails with code 550. CentOS 7.2

### ISSUE THREAD

**Issue by heinekenstar** [duplicati/duplicati#1885]
Hi Sir,
During duplicati version 2.0.1.17, I can backup my file to FTP Server and restore successfully,
but after upgrade to 2.0.1.18, the same backup job can not restore anymore with error code 550,
![2 0 1 18](https://cloud.githubusercontent.com/assets/20985569/17615650/39a4901a-60a2-11e6-8c68-988d03536e3c.png)
neither my duplicati client machine (CentOS 7.2.1511 & mono  4.4.2) nor my ftp server's config has changed,
and during duplicati version 2.0.1.18, I add a new backup job back up to FTP , when I press Test connection button it comes to the same issue, Failed to connect: Server returned an error: 550 Failed to change directory
![2 0 1 18 backup to ftp test connection](https://cloud.githubusercontent.com/assets/20985569/17615748/c6afa6b6-60a2-11e6-9f7a-d916404c0d09.png)

I've checked that my Duplicati client machine, it can ftp to FTP Server and access the folder correctly,
the same setting goes well on 2.0.1.17.
thank you.
##

**Comment by kenkendk** [comment]
There was a few minor tweaks to the UI from 2.0.1.17 to .18.
Could you check if there is a difference in the URLs generated?
Maybe a trailing slash or similar?

**Comment by heinekenstar** [comment]
@kenkendk thank you for your comment,
I already checked that whether one slash or two slash between FTPserver and Path can't backup correctly,
for example:
ftp://10.0.0.1:21//folder or ftp://10.0.0.1:21/folder
(by default, the URLs that Duplicati generate makes two slash between "Server and port" and "Path on server", maybe it's a bug too, but it did work fine at 2.0.1.17)

**Comment by aw115** [comment]
Just a note on this -- I'm having the same problem.  Connects fine when I go back to 1.3.4, but the latest release I get the same error.  Was there a fix to this, or one planned? Would love to see this working.

**Comment by ts678** [comment]
There's only one report on the Internet that Google can find of a "duplicati" "550 Failed to change directory", so this seems odd.
I looked for relevant code change in FTP between 2.0.1.17 and 2.0.1.18, but didn't find any. As said, change might be elsewhere.
I also checked changelog, and GitHub history. There have been changes, but nothing was talking specifically about this problem.
My `Test connection`, backup, and restore work (although I'm not using CentOS 7.2 -- this is Windows going to Filezilla Server.
Maybe something was fixed? The reports have stopped.

So -- unless somebody is still interested in this issue and is seeing it with a current release, the issue will close by itself eventually.
